### DISCUSSION TITLE
How to make v-col fill the remaining width

### DISCUSSION THREAD

**Discussion by Gianthard-cyh** [vuetifyjs/vuetify#13180] [Q&A] [answered]
I'm trying to place a v-textfield and a v-btn into the same row.
However, I got this:
![image](https://user-images.githubusercontent.com/45843411/109414191-90972780-79ec-11eb-8745-5ac5c9a62f71.png)
![image](https://user-images.githubusercontent.com/45843411/109414224-c4724d00-79ec-11eb-84a9-ad04d123e60e.png)

I just want to kick the button to the right side without changing its width, and the v-textfield will fill the width remaining.
![image](https://user-images.githubusercontent.com/45843411/109414264-000d1700-79ed-11eb-800e-4393b8457835.png)

**Comment by Kugeleis** [comment]
You could use the justify property of the grid system https://vuetifyjs.com/en/components/grids/#justify

**Reply by alielmajdaoui** [reply, marked as answer]
Use `cols="auto"` for the column that you want its width to be equal with its content width
```
<v-row dense>
  <v-col>
    <v-text-field label="Label" hide-details="auto" />
  </v-col>
  <v-col cols="auto">
    <v-btn color="info">Button</v-btn>
  </v-col>
</row>
```

[Live example](https://play.vuetifyjs.com/#eNp9ULtOxDAQ/JXFTUAipEc5JKgo6CkIhS/egKWNbdnrA+l0/87a5hFdQeHHjmdnvPNyVCnOw30IN4eM6laNjGsgzXg3OYDx0OsQ6rUWs3esrcP4DVUw+g8w6FJr+YVnTxugQoyf3C8WyQDpPdJuUk/lnBS8W4O9QVGnJLDO7AUdtpLDuWYzAVl/LeeWe3aF4KMwrFsK4yEze1fk5PEfg1LLbD/T19ft+AVp6YzDJjUp0xxtYEjIuYVn1+AjwxEiLnCCJfoVOgm8K3SQD7rEsKY32BXGZfeIRB6efSRz0V0VgyYpYup07TJR3V6/AHPPixg=)

![image](https://github.com/user-attachments/assets/28abb46b-91e0-4478-b978-d3885af909fb)
